src/app/organization/offices/create-offices/create-offices.component.html

```diff
@@ -0,0 +1,48 @@
+<div class="container">
+  <mat-card>
+    <form [formGroup]="newOfficeForm" (ngSubmit)="submit()">
```

Comment:
```suggestion
    <form [formGroup]="officeForm" (ngSubmit)="submit()">
```
Naming convention should be consistent with the other components of the web app. Make the required changes.